pycfmodel/model/resources/properties/security_group_ingress_prop.py

```diff
@@ -30,12 +33,20 @@ class SecurityGroupIngressProp(Property):
     SourceSecurityGroupOwnerId: Optional[ResolvableStr] = None
     ToPort: Optional[ResolvableInt] = None
 
+    @validator("CidrIp", pre=True)
+    def set_CidrIp(cls, v):
+        return IPv4Network(v, strict=False)
```

Comment:
Make sure it deals with resolvables too

Reply:
Updated resolver test to show it using `strict=False`